src/codeu/chat/client/commandline/Chat.java

```diff
@@ -238,17 +280,17 @@ public void invoke(Scanner args) {
     //
     panel.register("c-add", new Panel.Command() {
       @Override
-      public void invoke(Scanner args) {
-        final String name = args.hasNext() ? args.nextLine().trim() : "";
-        if (name.length() > 0) {
+      public void invoke(List<String> args) {
+        if(args.size() != 1){
+          System.out.println("ERROR: Failed to add new user");
+        }
+        else{
+          String name = args.get(0);
           final ConversationContext conversation = user.start(name);
           if (conversation == null) {
             System.out.println("ERROR: Failed to create new conversation");
-          } else {
-            panels.push(createConversationPanel(conversation));
           }
-        } else {
-          System.out.println("ERROR: Missing <title>");
+            panels.push(createConversationPanel(conversation));
```

Comment:
We still need else in here, we won't create the panel unless the conversation is not null.